src/automation/azext_automation/aaz/latest/automation/hrwg/__cmd_group.py

```diff
@@ -1,23 +1,23 @@
-# --------------------------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-#
-# Code generated by aaz-dev-tools
-# --------------------------------------------------------------------------------------------
-
-# pylint: skip-file
-# flake8: noqa
-
-from azure.cli.core.aaz import *
-
-
-@register_command_group(
-    "automation hrwg",
-)
-class __CMDGroup(AAZCommandGroup):
-    """Automation Hybrid Runbook Worker Group
-    """
-    pass
-
-
-__all__ = ["__CMDGroup"]
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "automation hrwg",
+)
+class __CMDGroup(AAZCommandGroup):
+    """Hrwg
+    """

```

Comment:
Why do you replace the full name in the comments with an abbreviation? In fact, the full name `Automation Hybrid Runbook Worker Group` is more readable and easier for others to understand than `Hrwg`?

Reply:
Reverted